src/utils/column-helper.ts

```diff
@@ -46,6 +46,10 @@ export function setColumnDefaults(columns: TableColumn[]) {
       column.draggable = true;
     }
 
+    if(!column.hasOwnProperty('draggable')) {
```

Comment:
This is a typo it should say 'if(!column.hasOwnProperty('isGroup')) {'

corrected.